app/connectors/aws/S3EventParser.scala

```diff
@@ -77,7 +77,7 @@ class S3EventParser @Inject()(implicit ec: ExecutionContext) extends MessagePars
         // use MdcLoggingExecutionContext / LoggingDetails.
         try {
           MDC.put("file-reference", event.location.objectKey)
```

Comment:
Do you have access to LoggingDetails?

Reply:
This refers to an outdated version.  This direct use of MDC was removed in a later commit, as it was adding the wrong value as the 'fileReference'.  A new objectKey is allocated when verify moves the file post scan, and so objectKey != the original upload fileReference.